.env.example

```diff
@@ -110,6 +110,9 @@ GITHUB_TOKEN=
 
 
 ## Jenkins, optional, for triggering Jenkins builds after deployment
+## When JENKINS_NOTIFY_COMMITTERS is set, it will send out emails of deployer, buddy-approver ( if available ), and committers
+## to jenkins as "emails" param. If GOOGLE_DOMAIN is set, then emails are filtered by domain.
 # JENKINS_URL= # server_url of jenkins
 # JENKINS_USERNAME= # user id
 # JENKINS_API_KEY= # API Token from user / Configure page
+# JENKINS_NOTIFY_COMMITTERS=1 #optional
```

Comment:
also not what the code does ...

`# optional: also send committers to jenkins for notiification in 'emails' param (filtered by GOOGLE_DOMAIN)

should there be a separate JENKINS_NOTIFY=1 flag ?